scripts/quests/otherAreas/Tango_with_a_Tracker.lua

```diff
@@ -54,6 +54,7 @@ quest.sections =
             {
                 [576] = function(player, csid, option, npc)
                         npcUtil.giveKeyItem(player, xi.ki.LETTER_FROM_SHIKAREE_X)
+                        player:addGil(10000)
```

Comment:
There is an npcUtil to give gil 

npcUtil.giveCurrency(player, "gil", xi.settings.main.GIL_RATE * 10000)

Reply:
> there is also a call to give players 10k gil in the battlefield code, although I have no idea if that part is bugged or not https://github.com/AirSkyBoat/AirSkyBoat/blob/staging/scripts/zones/Boneyard_Gully/bcnms/tango_with_a_tracker.lua

Typically, in my experience, the gil was added it just didnt give the message. The message would have to come from something like 
        player:addGil(xi.settings.main.GIL_RATE * 120)
        player:messageSpecial(ID.text.GIL_OBTAINED, xi.settings.main.GIL_RATE * 120)